lib/parsec/jinja2support.py

```diff
@@ -95,7 +95,9 @@ def jinja2environment(dir_=None):
     env = Environment(
         loader=ChoiceLoader([FileSystemLoader(dir_), PyModuleLoader()]),
         undefined=StrictUndefined,
-        extensions=['jinja2.ext.do'])
+        extensions=['jinja2.ext.do'],
+        autoescape=True
```

Comment:
Probably not the setting you want enabled for the suite templating:
```
{% set task_chain = 'foo => bar => baz' %}
{% autoescape True %}
graph = {{task_chain}}
{% endautoescape %}
```
outputs:
```
graph = foo =&gt; bar =&gt; baz
```

See also this in regard to enabling it globally by default: http://jinja.pocoo.org/docs/2.10/faq/#why-is-autoescaping-not-the-default